php-81/Dockerfile

```diff
@@ -95,200 +86,6 @@ RUN set -xe; \
  && rm ${INSTALL_DIR}/lib/libz.a
 
 
-###############################################################################
-# OPENSSL
-# https://github.com/openssl/openssl/releases
-# Needs:
-#   - zlib
-# Needed by:
-#   - curl
-#   - php
-RUN yum install -y perl-IPC-Cmd
-ENV VERSION_OPENSSL=3.0.10
-ENV OPENSSL_BUILD_DIR=${BUILD_DIR}/openssl
-ENV CA_BUNDLE_SOURCE="https://curl.se/ca/cacert.pem"
-ENV CA_BUNDLE="${INSTALL_DIR}/bref/ssl/cert.pem"
-RUN set -xe; \
-    mkdir -p ${OPENSSL_BUILD_DIR}; \
-    curl -Ls https://github.com/openssl/openssl/releases/download/openssl-${VERSION_OPENSSL}/openssl-${VERSION_OPENSSL}.tar.gz \
-  | tar xzC ${OPENSSL_BUILD_DIR} --strip-components=1
-WORKDIR  ${OPENSSL_BUILD_DIR}/
-RUN CFLAGS="" \
-    CPPFLAGS="-I${INSTALL_DIR}/include -I/usr/include" \
-    LDFLAGS="-L${INSTALL_DIR}/lib64 -L${INSTALL_DIR}/lib" \
-    ./config \
-        --prefix=${INSTALL_DIR} \
-        --openssldir=${INSTALL_DIR}/bref/ssl \
-        --release \
-        enable-tls1_3 \
-        no-tests \
-        shared \
-        zlib
-# Explicitly compile make without parallelism because it fails if we use -jX (no error message)
-# I'm not 100% sure why, and I already lost 4 hours on this, but I found this:
-# https://github.com/openssl/openssl/issues/9931
-# https://stackoverflow.com/questions/28639207/why-cant-i-compile-openssl-with-multiple-threads-make-j3
-# Run `make install_sw install_ssldirs` instead of `make install` to skip installing man pages https://github.com/openssl/openssl/issues/8170
-RUN make -j1 install_sw install_ssldirs
-RUN mkdir -p ${INSTALL_DIR}/bref/ssl && curl -Lk -o ${CA_BUNDLE} ${CA_BUNDLE_SOURCE}
-
-
-###############################################################################
-# LIBXML2
-# https://github.com/GNOME/libxml2/releases
-# Uses:
-#   - zlib
-# Needed by:
-#   - php
-#   - libnghttp2
-ENV VERSION_XML2=2.11.5
```

Comment:
Native version: 2.10.4

Reply:
We do backport security fixes to the version we ship in Amazon Linux. See https://alas.aws.amazon.com/AL2023/ALAS-2023-343.html for an example of this for `libxml2`.